lib/studies/rally/app.dart

```diff
@@ -22,13 +23,24 @@ class RallyApp extends StatelessWidget {
   static const String loginRoute = '/rally/login';
   static const String homeRoute = '/rally';
 
+  final sharedZAxisTransitionBuilder = const SharedAxisPageTransitionsBuilder(
+    fillColor: RallyColors.primaryBackground,
+    transitionType: SharedAxisTransitionType.scaled,
+  );
+
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       title: 'Rally',
       debugShowCheckedModeBanner: false,
       theme: _buildRallyTheme().copyWith(
         platform: GalleryOptions.of(context).platform,
+        pageTransitionsTheme: PageTransitionsTheme(
```

Comment:
Would this also change the transition for going to other pages in Rally? I guess you already created a custom transition for going into an account page